Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM registry.redhat.io/ubi8/python-312:1
```

Comment:
A small nitpick, but if you use `registry.access.redhat.com` instead of `registry.redhat.io` for UBI images, you don't need authentication to pull it down. It just simplifies things.

Reply:
you're right, this was updated here: https://github.com/opendatahub-io/ilab-on-ocp/pull/262/files